### PR TITLE
WIP: 95 conclude metadata design documentation

### PR DESCRIPTION
**Pull request by longshuicy**
Might include more UI screenshot once the definition part is ready

### CODE REVIEW

File: backend/docs/source/metadata.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+
+## Structure of Metadata Definition
+The structure of metadata definition looks like below:
+```
+{
+    "name" : "Unit",
+    "description" : "Unit",
+    "context" : {
+        "doi" : "https://schema.org/QuantitativeValue"
```
Comment (arunapa) [inline backend/docs/source/metadata.md:17]: Should be "unit" instead of doi

**Comment by lmarini** [comment]
can we merge this?

**Comment by longshuicy** [comment]
Sure let's merge this

**Comment by longshuicy** [comment]
> can we merge this?

Yes you can merge this

**Pull request merged**